fedmsg_meta_fedora_infrastructure/scm.py

```diff
@@ -56,15 +57,17 @@ def long_form(self, msg, **config):
             except KeyError:
                 repo = '.'.join(msg['topic'].split('.')[5:-1])
 
+            namespace = msg['msg']['commit'].get('namespace', 'rpms')
```

Comment:
Nice, I think this will also take care of #434. Can you amend the commit message to note that you're fixing that issue, too?